adoc/MAIN.release-notes.adoc

```diff
@@ -26,39 +42,41 @@ This release supports deployment on
 
 //Main features of Beta 4: Dex/Gangway, Supportconfig, k8s 1.15
 
-== Changes To Base OS
+== What is New
 
-=== Base Operating System Is Now {slsa} 15 SP1
+=== Changes To Base OS
```

Comment:
Is this extra headline still necessary? Seems you could do without and flatten structure.